e2e/cypress/integration/channel_sidebar/more_unreads_position_with_scroll_spec.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel_sidebar
+
+describe('Channel sidebar', () => {
+    let mainUser;
+    let otherUser;
+    let myTeam;
+
+    // This will ensure a list of channel names that will be alphabetically sorted
+    const channelNames = new Array(20).fill(1).map((value, index) => `scroll${index}`);
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user: firstUser}) => {
+            mainUser = firstUser;
+            myTeam = team;
+
+            cy.apiCreateUser().then(({user: secondUser}) => {
+                otherUser = secondUser;
+                return secondUser.id;
+            }).then((userId) => cy.apiAddUserToTeam(team.id, userId));
+
+            cy.wrap(channelNames).each((name) => {
+                const displayName = `channel-${name}`;
+                cy.apiCreateChannel(team.id, name, displayName, 'O', '', '', false).then((response) => {
+                    const testChannel = response.body;
+                    cy.apiAddUserToChannel(testChannel.id, mainUser.id);
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+        });
+    });
+
+    it('should display "More Unreads" pill at one of extremities of the sidebar when scrolling is active', () => {
```

Comment:
We are mapping written script against the test management to easily determine automated and manual tests coverage. That's why we're adding key and name in the it/test block.
```suggestion
    it('MM-T888 Channel sidebar: More unreads', () => {
```